translations/README.bn.md

```diff
@@ -0,0 +1,101 @@
+| 15 | [উৎপাদন](./4-manufacturing) | Train a fruit quality detector | ক্লাউডের ছবি শ্রেণিবদ্ধকরণ মডেলকে (Image Classifier) ফলের মান সনাক্ত করতে  কীভাবে প্রশিক্ষিত করতে হবে সে সম্পর্কে জানা | [Train a fruit quality detector](./4-manufacturing/lessons/1-train-fruit-detector/README.md) |
+| 16 | [উৎপাদন](./4-manufacturing) | Check fruit quality from an IoT device | আইওটি ডিভাইসে ফলের গুণগত মান সনাক্তকারী ব্যবহার | [Check fruit quality from an IoT device](./4-manufacturing/lessons/2-check-fruit-from-device/README.md) |
+| 17 | [উৎপাদন](./4-manufacturing) | Run your fruit detector on the edge | ফলের গুণগত মান সনাক্তকারীকে Edge হিসেবে ব্যবহার | [Run your fruit detector on the edge](./4-manufacturing/lessons/3-run-fruit-detector-edge/README.md) |
+| 18 | [উৎপাদন](./4-manufacturing) | Trigger fruit quality detection from a sensor | সেন্সর থেকে ফলের গুণাগুণ সনাক্তকরণ নিয়ন্ত্রণ করা শেখা| [Trigger fruit quality detection from a sensor](./4-manufacturing/lessons/4-trigger-fruit-detector/README.md) |
+| 19 | [খুচরাপর্যায়](./5-retail) | Train a stock detector | কোনও দোকানে স্টক গণনা করতে স্টক ডিটেক্টরকে প্রশিক্ষণ দেওয়ার জন্য কীভাবে অবজেক্ট সনাক্তকরণ ব্যবহার করা যায় তা শেখা | [Train a stock detector](./5-retail/lessons/1-train-stock-detector/README.md) |
+| 20 | [খুচরাপর্যায়](./5-retail) | Check stock from an IoT device | কোন অবজেক্ট সনাক্তকরণ মডেল ব্যবহার করে আইওটি ডিভাইস থেকে স্টক পর্যবেক্ষণ করা শেখা  | [Check stock from an IoT device](./5-retail/lessons/2-check-stock-device/README.md)  |
+| 21 | [খুচরাপর্যায়](./6-consumer) | Recognize speech with an IoT device | আইওটি ডিভাইস থেকে বক্তব্য (speech) সনাক্ত করে স্মার্ট টাইমার তৈরী করা | [Recognize speech with an IoT device](./6-consumer/lessons/1-speech-recognition/README.md) |
+| 22 | [ভোক্তাপর্যায়](./6-consumer) | Understand language | আইওটি ডিভাইসকে কীভাবে কথা বোঝাতে হয় তা শেখা | [Understand language](./6-consumer/lessons/2-language-understanding/README.md) |
+| 23 | [ভোক্তাপর্যায়](./6-consumer) | Set a timer and provide spoken feedback | আইওটি ডিভাইসে কীভাবে টাইমার সেট করতে হয় এবং টাইমার কখন সেট হয় এবং তা কখন শেষ হয় সে বিষয়ে কথিত  প্রতিক্রিয়া যেন সেই আইওটি ডিভাইস জানাতে পারে তা শেখা | [Set a timer and provide spoken feedback](./6-consumer/lessons/3-spoken-feedback/README.md) |
+| 24 | [ভোক্তাপর্যায়](./6-consumer) | Support multiple languages | কীভাবে আইওটি ডিভাইসে নির্দেশ দেয়া এবং স্মার্ট টাইমার থেকে আসা প্রতিক্রিয়া উভয়েই একাধিক ভাষা সাপোর্ট করানো যায় তা শেখা | [Support multiple languages](./6-consumer/lessons/4-multiple-language-support/README.md) |
+
+## অফলাইন ব্যবহার
+
+ [Docsify](https://docsify.js.org/#/) ব্যবহার করে পুরো ডকুমেন্টেশনটি অফলাইনে ব্যবহার করা যাবে। এজন্য এই রেপোসিটরি ফর্ক করতে হবে, তারপর লোকাল মেশিনে [Docsify ইন্সটল ](https://docsify.js.org/#/quickstart)করতে হবে। এবার এই রেপোসিটরির রুট ফোল্ডারে `docsify serve` লিখতে হবে। সবশেষে লোকাল হোস্টের পোর্ট ৩০০০ অর্থাৎ `localhost:3000` এ পুরো কোর্সটি ওয়েবসাইট আকারে পাওয়া যাবে।
```

Comment:
I've added a bit more to the end of the Readme since this was done. Can you update this to reflect the recent changes please.

Reply:
@jimbobbennett - The changes have been added.